openstack/content-repo/templates/cronjob.yaml

```diff
@@ -0,0 +1,85 @@
+{{- range $repo, $config := .Values.repos }}
+{{- if not $config.disabled }}
+apiVersion: batch/v1
+kind: CronJob
+
+metadata:
+  name: content-repo-{{ $repo }}
+  namespace: content-repo
```

Comment:
From the "let's fix this while we're at it" department: Namespace declarations on Helm templates are superfluous since Helm will override them with the target namespace anyway.